spec/integration/transaction_search_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'rails_helper'
+
+RSpec.describe "transaction search and filtering", type: :request do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.